qa/zcash/smoke_tests.py

```diff
@@ -128,28 +131,26 @@ def run_cmd(results, case, zcash, name, args=[]):
 
 def wait_for_balance(zcash, zaddr, expected=None, timeout=900):
     print('Waiting for funds to %s...' % zaddr)
-    if expected is not None:
-        print('Expecting %s' % expected)
-
-    unconfirmed_balance = zcash.z_getbalance(zaddr, 0)
-    print('Unconfirmed balance: %s' % unconfirmed_balance)
+    unconfirmed_balance = decimal.Decimal(zcash.z_getbalance(zaddr, 0)).quantize(decimal.Decimal('1.00000000'))
```

Comment:
Nit: I suggest importing Decimal directly so that it can be referred to more concisely.